base/abstractset.jl

```diff
@@ -98,8 +109,9 @@ end
     intersect(s, itrs...)
     ∩(s, itrs...)
 
-Construct the intersection of sets.
-Maintain order with arrays.
+Construct the intersection of sets. Maintain order with arrays.
```

Comment:
In the doctoring for `union` you changed the first sentence here. Would be good to be consistent.

Reply:
I am not sure what you mean here.